### DISCUSSION TITLE
Aeris Forecast 1,3 or 24

### DISCUSSION THREAD

**Discussion by jakesie1309** [poblabs/weewx-belchertown#760] [Q&A]
Hi there

I have enable forecast_interval_hours in the skin.conf  (forecast_interval_hours = 24), howerver this is not giving met the options of 1, 3 or 24. Just displays a normal 7 Day forecast.

Is there anything else i should do to get this to work.

**Comment by steepleian** [comment]
You need to be using the development version (1.3b1) for the functionality you require. The instructions can be found here: -

https://github.com/poblabs/weewx-belchertown#how-to-install-the-development-version

Ian

**Reply by swfrx** [reply]
This is missing from the latest release.  Is that deliberate, or simply an accidental omission?

**Comment by jakesie1309** [comment]
Hi Ian

Thank you so much. All working 100%

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
